T3697: fix the deadlock in IPsec options script

dmbaturin opened the pull request:
## Change Summary

An argument is passed to ipsec-settings.py when it's run from `vpn ipsec options` and IPsec is not fully configured yet, so that it doesn't try to restart StrongSWAN when it's not even supposed to be running yet.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

jestabro commented:
Discussed with Daniil; testing the version of this solution that was considered.

jestabro commented:
The PR (with import fixes, below) avoids the deadlock on boot when 'vpn ipsec options ..' is set; the 'version' referred to means that no other modifications are needed, for example, to vyatta-cfg-vpn.

Import corrections:
```
> @@ -18,7 +18,7 @@ import re
>  import os
>  
>  from time import sleep
> -from sys import exit
> +from sys import argv, exit
>  
>  from vyos.config import Config
>  from vyos import ConfigError
40c38
< +    res = system("ipsec stroke memusage >&/dev/null")
---
> +    res = os.system("ipsec stroke memusage >&/dev/null")
```

c-po commented:
@jestabro recently I found out that there is no need to import exit, its a keyword always available like if/for

dmbaturin committed:
T3697: do not try to restart charon if it's not required

The root cause is that the ipsec-settings.py script is run _twice_:
first from "vpn ipsec options", then from the top level "vpn" node.

The case when it's not required is when:
* "vpn ipsec" configuration doesn't exist yet
* user configured it with "vpn ipsec options"
* the ipsec-settings.py script is run first time, from "vpn ipsec options"

Trying to restart charon at that stage leads to a deadlock.

jestabro commented:
@c-po that's good to know --- the exit was there previously, but useful that it can be avoided; thanks !
@dmbaturin I'll run a sanity check on final version, and merge assuming all is fine

jestabro commented:
This revision looks good to me.
